### DISCUSSION TITLE
Having trouble using this module

### DISCUSSION THREAD

**Discussion by dtrinh100** [nuxt-modules/turnstile#326] [Q&A]
I am currently having trouble running this module. The token is always undefined. Am I doing something wrong? **Update**: looks like site-key for whatever reason is never passed to the NuxtTurnstile component, if you force it via a prop, it will work as expected.

My setup:

```
      <NuxtTurnstile ref="turnstile" v-model="token" />
      {{ token }}
```

nuxt.config.ts

```
export default defineNuxtConfig({
  runtimeConfig: {
    turnstile: {
      secretKey: "",
    },
  },

  turnstile: {
    siteKey: "my_actual_key",
  },

  modules: [
    "@nuxt/scripts",
    "@nuxtjs/turnstile",
  ],

});
```

package.json:
```
    "@nuxt/scripts": "^0.6.6",
    "@nuxtjs/turnstile": "^0.9.3",
```

**Comment by dargmuesli** [comment]
I think https://github.com/nuxt-modules/turnstile/issues/318 might cover your issue. Also see https://github.com/nuxt-modules/turnstile/pull/331 for a likely fix to the current stackblitz link if you want to provide a minimal reproduction.
